src/Command/AbsenceImportCSVCommand.php

```diff
@@ -144,14 +147,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         }
 
         $absences = array();
-        $db = new \db();
-        $db->select2('absences', null, array('cal_name' => 'hamac', 'perso_id' => "IN$ids_list"));
-        if ($db->result) {
-            foreach ($db->result as $elem) {
+        $abs= $this->entityManager->getRepository(Absence::class)->getByUserIds($ids_list, 'hamac');
```

Comment:
Ajouter un espace après $abs
`$abs = $this->entityManager->getRepository(Absence::class)->getByUserIds($ids_list, 'hamac');`